conf/lrm.conf

```diff
@@ -25,6 +25,7 @@ taiga	Taiga RISC-V core	https://gitlab.com/sfu-rcl/Taiga
 black-parrot	BlackParrot RISC-V core	https://github.com/black-parrot/black-parrot
 rsd	RSD RISC-V core	https://github.com/rsd-devel/rsd
 ivtest	Tests imported from ivtest	https://github.com/steveicarus/ivtest
+TNoC NoC router and fabric https://github.com/taichi-ishitani/tnoc
```

Comment:
Please make sure to separate the tag, the description and the url with tabs, otherwise it will not be parsed correctly

Reply:
I've fixed